projects/samples/contests/robocup/protos/Darwin-opHinge2Robocup.proto

```diff
@@ -1664,8 +1664,16 @@ PROTO Darwin-opHinge2Robocup [
                           endPoint Solid {
                             translation 0 -0.06 0.016
                             rotation 1 0 0 -2.0908
-                            name "handL[hand]"
+                            name "handL[arm]"
                             children [
+                              Transform {
+                                translation 0.0075 -0.1295 -0.016
+                                children [
+                                  Solid {
+                                    name "fingerL[hand]"
+                                  }
+                                ]
+                              }
```

Comment:
Adding an empty solid for the hand here seems odd...
Moreover, there is a name mismatch which doesn't look nice here: a hand is not an arm and a finger is not a hand...
Why can't we simply consider that a hand is part of an arm and handle the case this way?

Reply:
This choice is there to have a more realistic convex hull for the ball handling while keeping only 4 points. I agree that this is definitely not the cleanest solution, especially since the `[hand]` tag might be used for different things:

- Defining the property `armsLength`
- Convex hull for ball handling
- ?Ball manipulation?

For the last one I'm not sure if hand or arm is used, but in model specification it's rather defined according to the two previous points.

I think that the cleanest solution would be to rename the `[hand]` tag to `[arm_end]`, but I think that we currently need to avoid making changes to robot specifications if they are not required.

My main concern here is from a functional point of view, would it be harmful for the current implementation of the AutoReferee?

Reply:
> For the last one I'm not sure if hand or arm is used, but in model specification it's rather defined according to the two previous points.

Both are used.

> My main concern here is from a functional point of view, would it be harmful for the current implementation of the AutoReferee?

Anything that makes the referee track more solid positions will affect the performance. We are however working on some optimization that will make position tracking more efficient.